CONTRIBUTING.md

```diff
@@ -62,3 +63,47 @@ variable:
 ```bash
 $ export EXERCISES_API_URL=http://localhost:9292
 ```
+
+### Track Anatomy
+
+Each track should have the following structure:
+
+```bash
+├── .gitignore
+├── .travis.yml
+├── LICENSE
+├── README.md
+├── SETUP.md
+├── bin
+│   └── fetch-configlet
+├── config.json
+├── docs
+│   ├── ABOUT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
+└── exercises
+    └── hello_world
+        ├── hello_world_example.file
+        ├── hello_world.file
+        └── hello_world_test.file
+```
+
+* `LICENSE` - The MIT License (MIT)
+* `README.md` - a thorough explanation of how to contribute to the track.
+* `SETUP.md` - ???
+* `bin` - scripts and other files related to running the track's tests, etc.
+* `config.json` - ???
```

Comment:
This has track-level configuration, such as which exercises (and in which order) are a part of the track, which exercises are deprecated, the track id, name of the language, location of the repository, and (optionally) the regex to recognize what files are part of the test suite (usually /test/i, but sometimes /spec/i and sometimes other things, if the test pattern is not included, then we'll assume /test/i).